<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitWindow</class>
 <widget class="QWidget" name="UnitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/BackButtonIcon.png</normaloff>:/BackButtonIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/MusicButtonIcon.jpg</normaloff>:/MusicButtonIcon.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingsButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/SettingsButtonIcon.png</normaloff>:/SettingsButtonIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="DataList">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>151</width>
     <height>280</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="PlantName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources/biobloom.qrc"/>
 </resources>
 <connections/>
</ui>
